common/meta/api/src/kv_api.rs

```diff
@@ -35,6 +37,39 @@ pub trait ApiBuilder<T>: Clone {
     async fn build_cluster(&self) -> Vec<T>;
 }
 
+/// Return a string that bigger than all the string prefix with input string(only support ASCII char).
+/// "a" -> "b"
+/// "1" -> "2"
+/// [96,97,127] -> [96,98,127]
+/// [127] -> [127, 127]
+/// [127,127,127, 127] -> [127,127,127, 127, 127]
+pub fn prefix_of_string(s: &str) -> common_exception::Result<String> {
```

Comment:
What's going on with this...

<img width="522" alt="image" src="https://user-images.githubusercontent.com/44069/178091818-bc0a57ce-662a-46dd-a4a3-d3bb56848805.png">